blog/source/run-desktop-applications-in-hyper_.md

```diff
@@ -0,0 +1,91 @@
+title: "Run Desktop Applications in Hyper_"
+date: 2016-06-07 20:00:00 +0800
+author: hyper
+preview: Hyper_ cloud is designed to work well with many kinds of applications. That's why most people use it to deploy their applications in a cloud environment. This is great, and saves people huge headaches, but this article will show you a not-so-typical way.
+
+---
+
+In this article, we will show how to use Hyper_ to help run desktop applications on the cloud. The basic idea is that we will launch a Linux desktop version OS on Hyper_ cloud and then we can launch Chrome, Firefox etc in that container by using VNC.
+
+> More about VNC: VNC is a great client/server tool to access your computers remotely. Please check [tigervnc](http://tigervnc.org/) for example if you are not familiar with it. 
+
+Wait, why would we even want to run all these applications in Hyper_? 
+The most common case is that users may not want to install all the things on their PCs. 
+They may also want the ability to delete an application and know it is gone without some random file hanging around. 
+Or, they may just want a machine lives in the cloud!
+
+Obviously, running those applications in Hyper_ cloud will help a lot. What's more, now users can control how much CPU and Memory the application uses, and visit this application anywhere as long as they have Internet. 
+
+Not like sharing kernel in OS containers, Hyper_ leverage independent Linux Kernel for every container, that makes running desktop application in Hyper_ much easier, let's see how to do!
```

Comment:
Independent kernel seems to be a different story to tell (for example, when your app tries read from `/proc` or tuning kernel), so I will delete this part. Thanks!